Package.resolved

```diff
@@ -1,6 +1,15 @@
 {
   "object": {
     "pins": [
+      {
```

Comment:
This was generated when running `make docker_test`

Reply:
Please revert, as CyptoSwift is only used on Linux.